docs/baking.md

```diff
@@ -0,0 +1,159 @@
+download large `sprout-groth16.params` which is unused at the runtime.
+So, it's better to download them manually to the `/usr/local/share/zcash-params`, in order
+to do that run the following commands:
+```
+sudo mkdir -p /usr/local/share/zcash-params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-output.params -O /usr/local/share/zcash-params/sapling-output.params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-spend.params -O /usr/local/share/zcash-params/sapling-spend.params
+```
+
+## Setting up the tezos-node
+
+### Bootstrapping the node
+
+In order to run baker locally, you'll need fully-synced local `tezos-node`.
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `delphinet` or `edonet`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package. However, by default, these services will start to bootstrap the node
+from scratch, which will take significant amount of time. The fastest way to bootstrap
+the node is to import a snapshot. The snapshots can be downloaded from the https://snapshots.tulip.tools/#/ .
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as a an RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+All commands within the service are run under `tezos` user.
+
+Download the snapshost for desired network and with desired history mode
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires `tezos-node` data directory doesn't contain `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+In case you're getting the error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do the run the following:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
```

Comment:
I suppose these aren't interchangeable, right?

Wouldn't the `init` one always be necessary the first time around and the `update` one after that?

Reply:
Yes, only one of these two commands will work. It depends on whether the config was initialized previously by the node startup script.

Apparently, the wording should be changed to something like ''To do the run one of the following commands depending on whether the config in the given data directory was initialized previously:"